### DISCUSSION TITLE
全屏PCL2窗口

### DISCUSSION THREAD

**Discussion by YunzeWu** [Meloong-Git/PCL#2611] [提问] [answered]
请问龙猫有没有可能加入PCL2窗口最大化

**Comment by Pigeon0v0** [comment]
没必要

**Comment by WForst-Breeze** [comment, marked as answer]
#90 已拒绝

**Comment by Yue-cn** [comment]
用任务管理器全屏就知道为什么拒绝了

**Comment by Yue-cn** [comment]
对了，补充一下
![image](https://github.com/Hex-Dragon/PCL2/assets/79188080/0fe40392-b167-47dd-a76d-c96f5063d2f3)
